pom.xml

```diff
@@ -211,10 +218,18 @@
                 <version>2.13.3</version>
             </dependency>
 
+<!--            <dependency>-->
```

Comment:
This dependency can be removed, not just commented out.

Reply:
Removed from pom.xml